corehq/apps/userreports/reports/view.py

```diff
@@ -281,6 +282,117 @@ def lang(self):
             return self._lang
         return self.request.couch_user.language or default_language()
 
+    @property
+    def template_filters(self):
```

Comment:
nit: is it meant to be a list? did you mean `filters_template`?

Reply:
I think this is mirroring this code: https://github.com/dimagi/commcare-hq/blob/769ead9fbd17903be300e6910718594fd5781794/corehq/apps/reports/generic.py#L387

Reply:
okay, The naming threw me off.